README.md

```diff
@@ -1,155 +1,152 @@
 
-If you're looking for a more robust solution that comes with its own manager, I highly recommend [PlayEveryWhere's EOS Plugin for Unity](https://github.com/PlayEveryWare/eos_plugin_for_unity). The package is officially recommended by Epic and will quickly get your project set up if you don't care about fine tuning your implementation of the SDK. I also want to thank [Dylan Hunt's EOS-Unity project](https://gitlab.com/dylanh724/eos-unity) as it helped me understand the common init steps for EOS. I believe the project is deprecated now but still serves as a useful reference for getting set up with the SDK.
+If you're looking for a more robust solution that includes its own manager, I highly recommend [PlayEveryWhere's EOS Plugin for Unity](https://github.com/PlayEveryWare/eos_plugin_for_unity). The package is officially recommended by Epic and will quickly get your project set up if you don't care about fine-tuning your SDK implementation. I also want to thank [Dylan Hunt's EOS-Unity project](https://gitlab.com/dylanh724/eos-unity), which helped me understand the common initialization steps for EOS. I believe the project is deprecated now, but it still serves as a useful reference for setting up the SDK.
 
 ### How should I use OpenEOS?
 
-You can use OpenEOS in either or both of the following ways:
+You can use OpenEOS in either or both of the following ways:  
 
-1. A neatly packaged container for the EOS SDK that you can interface with directly by importing the Epic.OnlineServices namespace into your custom setup code.
-2. A set of extensions that themselves can interact with the packaged SDK which provide helper functions to let you more quickly get around the SDK. Simply import the RobProductions.OpenEOS namespace and you can use the *EOSCore*, *EOSAuth*, and *EOSEnv* static libraries to quickly implement some common SDK steps: Initialization, Shutdown, Auth system login, Connect system login, translating EpicAccountId to ProductUserId, and acquiring command line arguments passed from the EGS Launcher.
+1. A neatly packaged container for the EOS SDK that you can interface with directly by importing the `Epic.OnlineServices` namespace into your custom setup code.  
+2. A set of extensions that interact with the packaged SDK, providing helper functions to simplify SDK usage. Simply import the `RobProductions.OpenEOS` namespace to use the *EOSCore*, *EOSAuth*, and *EOSEnv* static libraries to quickly implement common SDK steps: initialization, shutdown, authentication system login, connect system login, translation of `EpicAccountId` to `ProductUserId`, and acquisition of command-line arguments from the EGS Launcher.
```

Comment:
"namespace to use" ... "to quickly implement" is a bit odd, maybe we can change this to "namespace to import the *EOSCore*" etc. and "static libraries which let you quickly implement common SDK steps"

Also "Auth" and "Connect" are key words in EOS that represent systems, would prefer to leave them as is